libcflib/db.xsh

```diff
@@ -43,7 +43,7 @@ class DB:
                 git pull $LIBCFGRAPH_URL master
         else:
             LOGGER.log('grabbing initial graph', category="db")
-            git clone --quiet $LIBCFGRAPH_URL $LIBCFGRAPH_DIR
+            git clone --depth=1 --quiet $LIBCFGRAPH_URL $LIBCFGRAPH_DIR
```

Comment:
I think this caused problems in the past. I don't remember what they were though.

Reply:
This change was requested by @mariusvniekerk

Reply:
Ah ok, then I guess it isn't a problem then!

Reply:
Ahh, I mean that @mariusvniekerk asked for the removal of `--depth=1`